src/lib.rs

```diff
@@ -99,6 +99,12 @@ impl<L: LogLevel> Verbosity<L> {
         }
     }
 
+    /// Whether any verbosity flags (either `--verbose` or `--quiet`)
+    /// have been specified on the command line.
+    pub fn is_set(&self) -> bool {
```

Comment:
I think `is_present` would be more consistent with how the clap API refers to this case.

Reply:
https://docs.rs/clap/latest/clap/struct.ArgMatches.html#method.args_present

Looks to be so. I will change the name.